package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bundled-es-modules/cookie",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
the original idea was that the version here needs to follow the version of the original packages, but it's not the case here and too late to fix

I'm improvising, in this case I tried to check for backwards compatibility and see 2.0.1 fit given the previous version bundled was 0.5.x (hard to say which exactly given the lack of the lock file before, see this PR https://github.com/bundled-es-modules/cookie/pull/3)